docs/source/v1.9.md.inc

```diff
@@ -18,9 +18,10 @@
 
 [//]: # (- Whatever (#000 by @whoever))
 
-[//]: # (### :bug: Bug fixes)
+### :bug: Bug fixes
 
-[//]: # (- Whatever (#000 by @whoever))
+- Fixed incorrect inclusion of autobad channels when disabling `find_bad_channels_meg` for example after enabling it (#902 by @larsoner)
```

Comment:
```suggestion
- When running the pipeline with [`find_bad_channels_meg`][mne_bids_pipeline._config. find_bad_channels_meg] enabled, then disabling it and running the pipeline again, the report would incorrectly still contain the automatically detected bad channels from the first pipeline run. Now, we ensure that the respective section is removed from the report in this case. (#902 by @larsoner)
```

Reply:
It's actually worse than this. The bads that were found automatically *would be included during processing* even after you disabled it, not just the report was misleading